examples/src/main/java/com/mapbox/navigation/examples/core/ReplayActivity.kt

```diff
@@ -78,14 +73,12 @@ class ReplayActivity : AppCompatActivity(), OnMapReadyCallback {
 
```

Comment:
Since the ReplayActivity is not directly related to freedrive. I'd prefer to take care of it outside of this PR. I have to look at all of the example activities anyway and make them consistent.